zimports/zimports.py

```diff
@@ -26,6 +27,12 @@
 from .vendored.flake8 import normalize_path
 
 
+class RewritePass(enum.Enum):
+    PLAIN = 0
+    TYPE_CHECK = 1
+    ANTI_TYPE_CHECK = 2
```

Comment:
just fyi, you can import `auto` from the enum module and use it to create the values here
```py
class RewritePass(enum.Enum):
    PLAIN = auto()
    TYPE_CHECK = auto()
    ANTI_TYPE_CHECK = auto()
```

Reply:
yeah I like the integers better at the moment

Reply:
btw auto would returns numbers.